model-optimizer/extensions/middle/RNNSequenceNormalizeToIE.py

```diff
@@ -154,14 +159,14 @@ def unsqueeze_num_directions(graph: Graph, match: dict):
             graph.remove_edge(rnn_layer.id, old_data_node.id)
             graph.add_edge(rnn_layer.id, data.id, key=0, out=i)
 
-            reshape = Reshape(graph, dict(dim=old_shape))
+            unsqueeze = Unsqueeze(graph, dict())
 
-            reshape_dim_data = Const(graph, {'name': rnn_layer.name + '/SqueezeNumDirections/{}/Dim'.format(i),
-                                             'value': old_shape}).create_node_with_data()
-            reshape.create_node_with_data([data, reshape_dim_data],
-                                          dict(name=rnn_layer.name + '/SqueezeNumDirections/{}'.format(i)),
-                                          data_nodes=[old_data_node])
+            unsqueeze_dim_data = Const(graph, {'name': rnn_layer.name + '/UnsqueezeNumDirections/{}/Dim'.format(i),
+                                             'value': [direction_dim[i]]}).create_node_with_data()
```

Comment:
It is not clear for me how the latest commit fixes the conversion issue of the MxNet model. Could you commen?